lib/triggers/csgostatTrigger.js

```diff
@@ -75,6 +75,9 @@ CSGOStatTrigger.prototype._respond = function(toId, userId, message) {
 		});
 		return true;
 	}
+	else if(query && params.length == 1) {
+		this._sendMessageAfterDelay(toId, 'Error reading command, use: ' + this.options.command + ' <steamID64>');
+		return false;
 	else {
```

Comment:
@andrewda There is an ending curly brace if you view the full source, I used it from the original else statement, that's why it's not appearing as changed.